src/routes/+page.svelte

```diff
@@ -1,7 +1,3 @@
-<script context="module">
-	export const prerender = true;
```

Comment:
Very good question, i wondered about that too during the conversion. Not sure what the best way is and not sure what you initially wanted for prerendering.

Reply:
perhaps this is not needed because we're using the static adapter – everything would be pre-rendered as a part of the build process right?

Reply:
yes, that's how i read the docs: https://kit.svelte.dev/docs/page-options#prerender

we also have it set to `true` in `src/routes/+layout.js` so it's set as default again.

i'll remove it.

Reply:
![Screen Shot 2023-01-23 at 4 19 22 PM](https://user-images.githubusercontent.com/498744/214152257-cdd96c1c-9a5c-4b10-9039-18dd8a041b09.png)
Ah yes, very good

Reply:
Setting it in one place seems to necessary. that's what i've done, kept it in `layout.js`.

```
@sveltejs/adapter-static: all routes must be fully prerenderable, but found the following routes that are dynamic:
    - src/routes/
  
  You have the following options:
    - set the `fallback` option — see https://github.com/sveltejs/kit/tree/master/packages/adapter-static#spa-mode for more info.
    - add `export const prerender = true` to your root `+layout.js/.ts` or `+layout.server.js/.ts` file. This will try to prerender all pages.
    - add `export const prerender = true` to any `+server.js/ts` files that are not fetched by page `load` functions.
```